src/EasyList/ParseAdd.cs

```diff
@@ -50,16 +50,44 @@ public static class ParseAdd
                 //processing for Label
                 positions.Add("add", (startIndex,endIndex - 1));
             }
+
             string label = GetData("add", args, positions).ToString().Trim();
-            string? description = positions.ContainsKey("-d") ? GetData("-d", args, positions).ToString() : string.Empty;
-            DateTimeOffset dueDate = positions.ContainsKey("-t") ? DateTimeOffset.Parse(GetData("-t", args, positions).ToString()) : DateTimeOffset.MaxValue;
-            TodoPriority priority = positions.ContainsKey("-p") ? Enum.Parse<TodoPriority>(GetData("-p", args, positions).ToString().ToUpper()) : TodoPriority.Low;
+
+            string? description;
+            if (positions.ContainsKey("-d"))
+            {
+                description = GetData("-d", args, positions).ToString();
+            }
+            else
+            {
+                description = string.Empty;
+            }
```

Comment:
I think it'd be good to change GetData to return a nullable, then do the following instead of all these if else statements.

`string description = GetData("-d", args, positions)?.ToString() ?? string.Empty;`

Possibly even make `GetData` return a string instead of string builder.